Gemfile

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 source 'https://rubygems.org'
+ruby '2.4.1'
```

Comment:
Putting version in README is also good.
I'm not sure editing Gemfile is good idea or not, especially when we develop gem.

By the way, should we remove Gemfile.lock...?